python/mxnet/gluon/data/vision/transforms.py

```diff
@@ -151,3 +151,23 @@ def __init__(self, max_brightness=0, max_contrast=0, max_saturation=0, max_hue=0
 
     def hybrid_forward(self, F, x):
         return F.image.random_color_jitter(x, *self._args)
+
+
+class AdjustLighting(HybridBlock):
```

Comment:
don't need this.

Reply:
OK. Then I will directly call the operator in the testing code